src/unity/python/turicreate/test/test_evaluation.py

```diff
@@ -979,12 +981,12 @@ def test_accuracy_multi_class_score(self):
 
         # Note: Explicitly not putting it into a for loop for ease of
         # debugging when the tests fail.
-        self.assertAlmostEqual(cls_skl_score[0], score[0])
-        self.assertAlmostEqual(cls_skl_score[0], str_score['0'])
-        self.assertAlmostEqual(cls_skl_score[1], score[1])
-        self.assertAlmostEqual(cls_skl_score[1], str_score['1'])
-        self.assertAlmostEqual(cls_skl_score[2], score[2])
-        self.assertAlmostEqual(cls_skl_score[2], str_score['2'])
+        self.assertAlmostEqual(prec_score[0], score[0])
```

Comment:
Why are you comparing this to precision score from sklearn. I'm a bit confused.

Reply:
I think the unit tests use sklearn as the source of truth. I just modified the existing unit tests to compare to precision. What would you recommend?